cookbook/serializer.rst

```diff
@@ -171,6 +171,15 @@ to your class and choose which groups to use when serializing::
         $someObject,
         'json', array('groups' => array('group1'))
     );
+    
+
+Instead of ``@Groups`` annotations, the Serializer component also supports Yaml
+or XML files. These configuration are automatically loaded from these locations:
```

Comment:
These files are automatically when being stored in one of the following locations: